samples/crypto/psa_tls/README.rst

```diff
@@ -1,13 +1,13 @@
 .. _crypto_tls:
 
-Crypto: PSA TLS
-###############
+Crypto: PSA Transport Layer Security
```

Comment:
The sample name is "psa_tls"
I was a bit confused trying to find the sample documentation in the doc preview link, before I realized the documentation was renamed.
"TLS" is a pretty established abbreviation, so I propose we keep the "abbreviated" sample name in the header to keep it simple for developers to match the doc with the sample name, and then we can expand the abbreviations in the text description